src/studio-home/__mocks__/studioHomeMock.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
     },
   ],
   librariesEnabled: true,
```

Comment:
I believe this scenarios will be checked as part of @rpenido 's [PR](https://github.com/open-craft/frontend-app-course-authoring/pull/37/files), since the placeholder is going to be swapped out to an actual page.